assets/js/components/Health/HealthIcon.test.jsx

```diff
@@ -32,4 +33,9 @@ describe('HealthIcon', () => {
     const svgEl = container.querySelector("[data-testid='eos-svg-component']");
     expect(svgEl.classList.toString()).toContain('hover:opacity-100');
   });
+  it('should display the icon with the applied size', () => {
+    const { container } = render(<HealthIcon health="passing" size="m" />);
+    const svgEl = container.querySelector("[data-testid='eos-svg-component']");
+    expect(svgEl).toHaveAttribute('width', '18');
+  });
```

Comment:
Keep in mind that here we are testing the implementation of an external library, so this is not really needed from my side. If you wanna keep it that's ok, but I'm not a huge fan of this

Reply:
Well, that's not entirely true. We are testing that the provided size value puts the correct icon size. So it tests the `size="m"` value, which is part of our component.
Later on, that we use that 3rd party element to assure that is other story